src/Operation/Aggregate.php

```diff
@@ -223,12 +204,6 @@ public function __construct(string $databaseName, ?string $collectionName, array
         $this->isExplain = ! empty($options['explain']);
         $this->isWrite = is_last_pipeline_operator_write($pipeline) && ! $this->isExplain;
```

Comment:
After these changes, `$this->isExplain` is only used on this line.

If you don't decide to add the condition I suggested later in `createCommandDocument()`, then perhaps we can just remove the property.

Reply:
Removed the property and inlined the usage - I also removed the `! empty` in favour of a null coalesce operator.